src/olympia/amo/utils.py

```diff
@@ -641,11 +641,17 @@ def resize_image(source, destination, size=None):
         original_size = im.size
         if size:
             im = processors.scale_and_crop(im, size)
-    with storage.open(destination, 'wb') as fp:
-        # Save the image to PNG in destination file path. Don't keep the ICC
-        # profile as it can mess up pngcrush badly (mozilla/addons/issues/697).
-        im.save(fp, 'png', icc_profile=None)
-    pngcrush_image(destination)
+    if not format:
+        with storage.open(destination, 'wb') as fp:
+            im = im.convert('RGB')
+            im.save(fp, 'JPEG')
```

Comment:
Default quality for `JPEG` format in Pillow is 75. That's certainly going to make the file size small, but maybe it's a bit too aggressive...

Reply:
I mostly tested the images currently in `static/img/hero/featured` that are already 960x640 jpgs. I think `quality` should be set to 80, as the majority of those file sizes remain the same upon upload with this setting. Thoughts?

Reply:
80 sounds good. We can bump to 90 if we notice issues with image quality.